contrib/dockerd-rootless.sh

```diff
@@ -105,5 +111,12 @@ else
 	# remove the symlinks for the existing files in the parent namespace if any,
 	# so that we can create our own files in our mount namespace.
 	rm -f /run/docker /run/containerd /run/xtables.lock
+
+	if [ -n "$_DOCKERD_ROOTLESS_SELINUX" ]; then
+		# iptables requires /run in the child to be relabeled. The actual /run in the parent is unaffected.
+		# https://github.com/containers/podman/blob/e6fc34b71aa9d876b1218efe90e14f8b912b0603/libpod/networking_linux.go#L396-L401
+		# https://github.com/moby/moby/issues/41230
+		chcon system_u:object_r:iptables_var_run_t:s0 /run
+	fi
```

Comment:
Wondering; should this be seen as a temporary approach (and ultimately be handled by rootlesskit or dockerd itself?)

Reply:
Ideally this could be fixed on the selinux policy package of RHEL/Fedora, however, given that even Podman has the same workaround, this workaround is likely to be permanent.

Reply:
I'd prefer to keep this in the script. 

Other distros/versions may have different policies for SELinux, and may require different chcon workarounds. The script is intended to be modifiable for such workarounds.